studio/stores/authConfig/schema/AuthProviders/AuthProvidersFormValidation.tsx

```diff
@@ -231,17 +231,17 @@ const PROVIDER_PHONE = {
       units: 'digits',
       show: {
         key: 'SMS_PROVIDER',
-        matches: ['twilio', 'messagebird', 'textlocal', 'vonage']
-      }
+        matches: ['twilio', 'messagebird', 'textlocal', 'vonage'],
+      },
     },
     SMS_TEMPLATE: {
       title: 'SMS Message',
-      type: 'string',
+      type: 'multiline-string',
```

Comment:
This is the actual change in this file. The rest is trailing commas added by prettier. I was going to remove them, but the rest of the file already has trailing commas so it felt OK to let it autoformat to bring it all in line.

Let me know if you want me to drop the formatting for a cleaner diff though!